src/data/__test-data.js

```diff
@@ -0,0 +1,143 @@
+// @flow
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Is that ESLint exception needed?

Reply:
Yeah, ESLint complains about it being a devDependency. I'm guessing it thinks this file is application code.

Reply:
Ah right. I see we already ignore other files here. So maybe we should do the same for this file?

https://github.com/redbadger/pride-london-app/blob/9ac60fe91e9239e40fd3a0fb027d454f69089498/.eslintrc#L40-L47